manifests/service.pp

```diff
@@ -0,0 +1,44 @@
+#
+define systemd::service (
+    $execstart,
+    $servicename     = $name,
+    $description     = '',
+    $execstop        = undef,
+    $execreload      = undef,
+    $workingdir      = undef,
+    $restart         = 'always',
+    $restartsec      = undef,
+    $remainafterexit = false,
+    $user            = 'root',
+    $group           = 'root',
+    $type            = 'simple',
+    $defaultdeps     = true,
+    $requires        = [],
+    $conflicts       = [],
+    $wants           = [],
+    $after           = [],
+    $wantedby        = ['multi-user.target'],
+    $unit_path       = $systemd::params::unit_path,
```

Comment:
In order for systemd::params::unit_path to work accesible the class must inherit systemd::params